dag/walkthrough.yml

```diff
@@ -25,4 +25,4 @@ steps:
   data://garden/ihme_gbd/2020-12-19/child_mortality:
   - data://meadow/ihme_gbd/2020-12-19/child_mortality
   data://grapher/ihme_gbd/2020-12-19/child_mortality:
-  - data://garden/ihme_gbd/2020-12-19/child_mortality
+  - data://garden/ihme_gbd/2020-12-19/child_mortality
```

Comment:
Maybe not for this PR, but should we create a separate dag file for `health` (or some other appropriate name) and move all these steps from `dag_walkthrough`?

Reply:
Good idea - I can do a separate PR for this